cfme/v2v/migration_plans.py

```diff
@@ -0,0 +1,598 @@
+                    self.in_migration_plan and self.plans_not_started_list.is_displayed
+                ),
+                "5.10": self.in_migration_plan and self.TITLE in self.title.read(),
+            }
+        )
+        return in_view
+
+
+class InProgressPlansView(MigrationPlanView):
+    TITLE = "In Progress"
+    progress_card = MigrationProgressBar(locator=".//div[3]/div/div[3]/div[3]/div/div")
+
+    @property
+    def is_displayed(self):
+        in_view = VersionPicker(
+            {
+                Version.lowest(): (self.in_migration_plan and self.progress_card.is_displayed),
+                "5.10": self.in_migration_plan and self.TITLE in self.title.read(),
+            }
+        )
+        return in_view
+
+
+class CompletedPlansView(MigrationPlanView):
+    TITLE = "Completed"
+    plans_completed_list = MigrationPlansList("plans-complete-list")
+
+    @property
+    def is_displayed(self):
+        in_view = VersionPicker(
+            {
+                Version.lowest(): (
+                    self.in_migration_plan and self.plans_completed_list.is_displayed
+                ),
+                "5.10": self.in_migration_plan and self.TITLE in self.title.read(),
+            }
+        )
+        return in_view
+
+
+class ArchivedPlansView(MigrationPlanView):
+    TITLE = "Archived"
+    migration_plans_list = MigrationPlansList("plans-archived-list")
+
+    @property
+    def is_displayed(self):
+        in_view = VersionPicker(
+            {
+                Version.lowest(): (
+                    self.in_migration_plan and self.migration_plans_list.is_displayed
+                ),
+                "5.10": self.in_migration_plan and self.TITLE in self.title.read(),
+            }
+        )
+        return in_view
+
+
+class AddMigrationPlanView(View):
+    title = Text(locator='.//h4[contains(@class,"modal-title")]')
+    name = TextInput(name="name")
+    description = TextInput(name="description")
+    back_btn = Button("Back")
+    next_btn = Button("Next")
+    cancel_btn = Button("Cancel")
+    form_title_text = VersionPicker(
+        {Version.lowest(): "Migration Plan Wizard", "5.10": "Create Migration Plan"}
+    )
+
+    @View.nested
+    class general(View):  # noqa
+        infra_map = BootstrapSelect("infrastructure_mapping")
+        name = TextInput(name="name")
+        name_help_text = Text(locator='.//div[contains(@id,"name")]/span')
+        description = TextInput(name="description")
+        select_vm = RadioGroup('.//div[contains(@id,"vm_choice_radio")]')
+
+        @property
+        def is_displayed(self):
+            return self.infra_map.is_displayed and self.name.is_displayed
+
+    @View.nested
+    class vms(View):  # noqa
+        import_btn = Button("Import")
+        importcsv_btn = Button("Import CSV")
+        hidden_field = HiddenFileInput(locator='.//input[contains(@accept,".csv")]')
+        # TODO: Replace string keys by integer keys after row indexing issue get fixed
```

Comment:
Please add `BZ` number or related `GH` issue for reference.

Reply:
We do not have such issue created as such, you can remove this `# TODO`. Since as per [this]( https://github.com/RedHatQE/widgetastic.core/blob/bfc352dd1856638b522e0f0953a8fdbaaa23cc5b/src/widgetastic/widget/table.py#L46) checks, from `widget` of `TableColumn`, self.column_name has some value that we didn't expect and that isn't visible to us and this get added to the column. This is similar issue like https://github.com/ManageIQ/integration_tests/pull/8295#discussion_r243289851.